conduit/test/test_resources.py

```diff
@@ -266,3 +266,47 @@ def test_gfk_foo_resource(self):
         self.assertEqual(content['integer'], 12)
         self.assertEqual(content['name'], 'Foo Name')
         self.assertEqual(content['text'], 'text goes here')
+
+    def test_related_fields_empty_data(self):
+        data = {
+            # 'bar': {},
+            # 'bazzes': {},
+            'boolean': False,
+            'decimal': '110.12',
+            'float_field': 100000.123456789,
+            'integer': 12,
+            'name': 'Foo Name',
+            'text': 'text goes here'
+        }
+        resource_uri = self.foo_resource._get_resource_uri()
+        response = self.client.post(
+            resource_uri,
+            json.dumps(data),
+            content_type='application/json'
+        )
+        content = json.loads(response.content.decode())
+        self.assertEqual(content['bar'], {})
```

Comment:
Currently the response from the api will return the key with an empty object, but maybe we don't want to have a non-required related field in the response at all if it doesn't exist?